marklogic-data-hub/src/test/java/com/marklogic/hub/entity/EntityManagerTest.java

```diff
@@ -191,6 +194,33 @@ public void overrideExistingExplorerOptions() {
         assertTrue(newStagingOptionsTimeStamp > oldStagingOptionsTimeStamp);
     }
 
+    @Test
+    public void deployExplorerOptionsWithoutContainerConstraint() {
+        Path entitiesDir = getHubProject().getHubEntitiesDir();
+        if (!entitiesDir.toFile().exists()) {
+            entitiesDir.toFile().mkdirs();
+        }
+        FileUtil.copy(getResourceStream("scaffolding-test/Collection.entity.json"),
+                entitiesDir.resolve("Collection.entity.json").toFile());
+
+        runAsDataHubDeveloper();
+        entityManager.deployQueryOptions();
+
+        runAsAdmin();
+        Stream.of(HubConfig.DEFAULT_STAGING_NAME, HubConfig.DEFAULT_FINAL_NAME)
+                .forEach(dbName -> assertNull(getModulesFile("/Default/" + dbName + "/rest-api/options/exp-default.xml"), "Did not expect exp-default.xml options file in " + dbName + "since we dont use it anymore."));
+
+        String stagingExplorerOptions = getModulesFile("/Default/" + HubConfig.DEFAULT_STAGING_NAME + "/rest-api/options/" + HubConfig.EXP_STAGING_ENTITY_QUERY_OPTIONS_FILE);
+        String finalExplorerOptions = getModulesFile("/Default/" + HubConfig.DEFAULT_FINAL_NAME + "/rest-api/options/" + HubConfig.EXP_FINAL_ENTITY_QUERY_OPTIONS_FILE);
+
+        Stream.of(stagingExplorerOptions, finalExplorerOptions).forEach(option -> {
```

Comment:
This is good, but the test needs to assert two more things - that the "default" options were written to src/main/entity-config in the project directory, and that they are not in the modules database because of the conflict between Collection constraints.

Reply:
Updated the tests.